### PR TITLE
Packages pushing to MyGet are invalid.

### PR DESCRIPTION
**Pull request by CodeBlanch**
This one took me on a journey today!

I consumed `OpenTelemetry.Collector.AspNet`, `OpenTelemetry.Collector.Dependencies`, & `OpenTelemetry.Exporter.Jaeger` from MyGet into a .NET 4.8 WebApplication.

On startup I would crash loading `OpenTelemetry.Exporter.Jaeger` because it was trying to load .NET Standard 2.1. That's weird, it should be loading .NET Standard 2.0. Project asset file looked good though, referencing `OpenTelemetry.Exporter.Jaeger` through the /lib/netstandard2.0/ path.

A bunch of Googling later, I decided to look at the assemblies in the package using a decompiler. Turns out the MyGet `OpenTelemetry.Exporter.Jaeger` nupkg /lib/netstandard2.0/ folder actually contains a binary targeting netstandard2.1!

I tracked this down to an issue in our MyGet pipeline. From [the last one](https://dev.azure.com/opentelemetry/pipelines/_build/results?buildId=1456&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=58b29dba-1c58-5441-f2a1-8b1882f87386):

```
Copying file from "D:\a\1\s\src\OpenTelemetry.Exporter.Jaeger\obj\Release\netstandard2.0\OpenTelemetry.Exporter.Jaeger.dll" to "D:\a\1\a\OpenTelemetry.Exporter.Jaeger.dll".
Copying file from "D:\a\1\s\src\OpenTelemetry.Exporter.Jaeger\obj\Release\netstandard2.1\OpenTelemetry.Exporter.Jaeger.dll" to "D:\a\1\a\OpenTelemetry.Exporter.Jaeger.dll".
```

Basically last target wins because the folder structure isn't capable of storing all the different outputs.

I took a stab at fixing it on this PR but I don't have a way to test the pipeline sorry!

### CODE REVIEW

**Comment by CodeBlanch** [comment]
@SergeyKanzhelev FYI New package looks good as far as including the correct binaries of the libs for each target! New issue: Jaeger package isn't including Thrift.dll or listing it as a dependency. I'll see if I can fix that.

**Comment by MikeGoldsmith** [comment]
Good investigation work @CodeBlanch :+1:
